packages/flow-client-react/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@speechmatics/flow-client-react",
```

Comment:
I see what you mean. My general thinking with the packages is that they have a particular function, rather than grouping things together too much under the SDK umbrella term. I.e. the `client` packages are framework/runtime agnostic clients for the APIs, while the react packages are react hooks on top of the clients.

Other packages we add in the future, like mic selection react hooks or audio utils, will probably be best named according to what they do as well. To me, the collection of all these tools is kind of what SDK means to me, rather than a particular package, but maybe my understanding of the term SDK isn't the correct one.

Reply:
That's a good point. On reflection I'd probably say your idea of an sdk makes most sense. In this case, the whole repo is the development kit, and the individual packages are just lightweight clients for individual segments - let's keep up with that!